src/components/EmojiMenuItem.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import BlockMenuItem, { Props as BlockMenuItemProps } from "./BlockMenuItem";
+import styled, { withTheme } from "styled-components";
+
+const Title = styled.p`
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  font-size: 16px;
+`;
+
+const EmojiTitle = ({
+  emoji,
+  title,
+}: {
+  emoji: React.ReactNode;
+  title: React.ReactNode;
+}) => {
+  return (
+    <Title>
+      {emoji}
```

Comment:
Can we wrap the emoji to give it 16px size but keep the text size the same as the other menu?

Reply:
ok ~